apps/player/src/app/watch/watch.component.ts

```diff
@@ -27,7 +28,7 @@ export class WatchComponent {
       const tokenId = this.route.snapshot.paramMap.get('getPlayerId') as string;
       const message = 'I confirm that I own this token';
       const signature = await this.metamaskService.signMessage(message);
-      this.playerUrl = await this.getPlayerId({ tokenId, message, signature }).toPromise();
+      this.playerUrl = await this.getPlayerId(createSignedMessage({ tokenId, message, signature })).toPromise();
```

Comment:
I don't see the point of using `createSignedMessage`, you've got already everything from above.